### DISCUSSION TITLE
Bypassing Android firewall

### DISCUSSION THREAD

**Discussion by ghost** [Mygod/VPNHotspot#517] [Q&A]
Hello there, 
First I wanna thank you for this app. It helped me in so many situations and for a long time... 

Now I wanna ask you if it's possible somehow to bypass the Android firewall. I'm trying to play online games on a console with an Android hotspot. I'm aware this is usually impossible because of the CG-NAT but my mobile provider has the possibility to assign a public IP on the phone. And I wanna forward this to my connected devices in the Android hotspot. I was trying the "IP Masquerade Mode" but that didn't really helped. I hosted a hotspot with this app and had "wlan1" turned on. Is there a way to make this working with this app? The public IP works on an iPhone perfectly so that's not the problem.

Thanks in advance and looking forward to hear from you.

**Comment by Mygod** [comment]
Try to use a port forwarding app?

**Reply by ghost** [reply]
That didn't work at all. I have really much tried. VPN Hotspot, Termux, VPN Hotspot, AFWall+, Fwd, Servers Ultimate... Nothing helps.
